helpers/hasCorrectNextConfig.js

```diff
@@ -12,9 +11,27 @@ const hasCorrectNextConfig = async ({ nextConfigPath, failBuild }) => {
   const isValidTarget = acceptableTargets.includes(target)
   if (!isValidTarget) {
     console.log(
-      `Your next.config.js must set the "target" property to one of: ${acceptableTargets.join(', ')}. Update the
-      target property to allow this plugin to run.`,
+      `The "target" config property must be one of "${acceptableTargets.join('", "')}". Setting it to "serverless".`,
```

Comment:
"Settingit to serverless" might imply to some that we're actually mutating/updating/changing their config file (which i don't think we're doing? 🤔 ). maybe "forcing the build to use target serverless" or something?

Reply:
Good point